From ac8faf2b14ed1c9ce518eddfc9c5098b6f9c74dd Mon Sep 17 00:00:00 2001
Message-Id: <ac8faf2b14ed1c9ce518eddfc9c5098b6f9c74dd.1523725229.git.jan.steffens@gmail.com>
From: Dylan Baker <dylan@pnwbakers.com>
Date: Wed, 4 Apr 2018 10:23:02 -0700
Subject: [PATCH 1/5] meson: fix megadriver symlinking

Which should be relative instead of absolute.

Fixes: f7f1b30f81e842db6057591470ce3cb6d4fb2795
       ("meson: extend install_megadrivers script to handle symmlinking")
Bugzilla: https://bugs.freedesktop.org/show_bug.cgi?id=105567
Signed-off-by: Dylan Baker <dylan.c.baker@intel.com>
Reviewed-and-Tested-by: Eric Engestrom <eric.engestrom@imgtec.com>
Reviewed-by: Emil Velikov <emil.velikov@collabora.com>
---
 bin/install_megadrivers.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/bin/install_megadrivers.py b/bin/install_megadrivers.py
index 86bfa35918..7931a544bd 100755
--- a/bin/install_megadrivers.py
+++ b/bin/install_megadrivers.py
@@ -58,7 +58,7 @@ def main():
             while ext != '.so':
                 if os.path.exists(name):
                     os.unlink(name)
-                os.symlink(driver, name)
+                os.symlink(each, name)
                 name, ext = os.path.splitext(name)
         finally:
             os.chdir(ret)
-- 
2.16.2

